docs/install/prerequisites.md

```diff
@@ -5,9 +5,8 @@
 One of the following:
 
 * BTF available under `/sys/kernel/btf/vmlinux` (see [libbpf CO-RE documentation] for more info).
-* Linux kernel headers available under conventional location (see [Linux Headers](./headers.md) section for more info).
+* Linux kernel headers available under conventional location (see [Linux Headers](../building/headers.md) section for more info).
   In this case system libraries libelf and zlib are required as well.
-* Tracee's eBPF probe pre-compiled (see [eBPF Compilation](./ebpf-compilation.md) section for more info).
```

Comment:
Why removing this line? We still support TRACEE_BPF_FILE

Reply:
Thats not the only change needed I'm afraid. We need a full review to pre-requisites (which I plan to do after Makefile.one is set to default). Not only preqrequisites to be honest, all documentation (and how to run containers).

Reply:
Yes, we probably should do it.
Anyway, we should keep this line